### DISCUSSION TITLE
Give Up Github?

### DISCUSSION THREAD

**Discussion by scriptiz** [plausible/analytics#1983] [Anything Else]
Just wondering if Plausible ever considered it?

https://giveupgithub.org

Personally I wonder if it would not impact badly the community and contributors to Plausible.

**Comment by metmarkosaric** [comment]
thanks for sharing @scriptiz! saw the discussion yesterday on HN too. some good points and worth thinking about. it won't be short term though as it's a big project and we already have a very long list of things to do so with our limited resources it's not easy to prioritize this.

another aspect that's not mentioned and that we hear about is that github is not easy to use for many people (non-developers). they're a big part of our audience. finding a more user friendly place for areas such as the feature requests and roadmap could be another thing to consider. then the code itself, self-hosted releases etc could live on some of those mentioned alternatives which are all kind of github-like in terms of design/usability (and probably just as difficult to use for newbies) while we keep the feature requests, roadmap and other stuff non-developers want in a different place

we do get some organic traffic from github as it's a very popular platform so how a potential move may affect our future growth is something to consider too. anyhow, again this is not something we have the capacity to handle in the short term but certainly something we will keep in mind for the future

thanks!

**Reply by scriptiz** [reply]
Makes total sense, better use the limited resources elsewhere for now :)

Thanks for your answer

**Reply by ukutaht** [reply]
> inding a more user friendly place for areas such as the feature requests and roadmap could be another thing to consider. then the code itself, self-hosted releases etc could live on some of those mentioned alternatives which are all kind of github-like in terms of design/usability (and probably just as difficult to use for newbies) while we keep the feature requests, roadmap and other stuff non-developers want in a different place

Yes I think this would be a good direction. Move our code to a platform that is actually open source (and not owned by Microsoft), while moving our feedback and roadmap tooling closer to our customers. I think it's important that our customers can access the feedback and roadmap areas without having to create an account on Github, for example.

In any case, if we do end up moving, we should still continue mirroring the project to Github since it also has a social and discovery aspect which other version control tools are not as strong on.
